addons/binding/org.openhab.binding.wso2iots/.idea/misc.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove intellij-specific stuff. BTW what is the reason for using intellij vs Eclipse? If this is just a matter of preference, I would suggest to keep it outside of this repo, as openHAB supports eclipse. Having two will most likely cause one or the other to become obsolete - for example other community users making changes will not update the intellij.